packages/clickable/README.md

```diff
@@ -0,0 +1,3 @@
+# Clickable Name
+
+Clickable documentation
```

Comment:
There seems to be documentation missing here

Reply:
Yes, I haven't written any documentation yet. :-/ And I should have done that before creating a PR.

Reply:
Or just leave it in WIP and setting the label ;)